adr/0002-quarkus-rest.adoc

```diff
@@ -0,0 +1,37 @@
+= Rename RESTEasy Reactive extensions to Quarkus REST
+
+* Status: proposed
+* Date: 2023-10-17
+
+== Context and Problem Statement
+
+RESTEasy Reactive is the preferred REST stack in Quarkus, however in the almost 3 years since its release, we have seen
+folks of all experience misunderstand its capabilities and being led to believe that it forces reactive programming on them.
```

Comment:
I would add something about virtual threads, too. I create confusion because most people consider reactive and virtual threads as opposite (while they are not - and even with lots of explanation, the bulldozer marketing from others is pretty hard to circumvent).